### PR TITLE
Update documentation

### LINKED ISSUE #55: Update Documentation to match completion handler changes
Completion handlers were changed as part of Issue #43, the documentation needs to be updated to match

### PR DESCRIPTION
**Pull request by rhyshort**
## What
Update the documentation to match the current APIs.

## Reviewers
reviewer @emlaver

## Issues
Fixes #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ db.add(operation:read)
 let delete = DeleteDocumentOperation()
 delete.docId = "doc1"
 delete.revId = "1-revisionidhere"
-delete.deleteDocumentCompletionHandler = {(statusCode, error) in
+delete.completionHandler = {(statusCode, error) in
```
Comment (emlaver) [inline README.md]: Shouldn't the example include the `httpInfo` and not contain `statusCode`? Example from a test case:
```
delete.completionHandler = {(response, httpInfo, error) in ...
```

Comment (rhyshort) [inline README.md]: yes it should, fixed in 0d17a87

File: README.md [hunk 2]
```diff
@@ -55,23 +55,23 @@ let db = client["database"]
...
+        NSLog("Created document \(response?["id"]) with revision id \(response?["rev"])");
     }
 }
 db.add(operation:create)
 
 // Read a document
 let read = GetDocumentOperation()
 read.docId = "doc1"
-read.getDocumentCompletionHandler = { (doc, error) in
+read.completionHandler = { (response, error) in
```
Comment (emlaver) [inline README.md]: What about updating the example of the get operation with the `httpInfo` parameter?

**Comment by emlaver** [comment]
LGTM +1

**Pull request merged**